api/db/changelog/db.changelog-6.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="blrubenstein" id="changelog-6">
+    <addColumn tableName="workspace">
+      <column name="rp_review_requested" type="boolean"/>
+      <column name="rp_approved" type="boolean">
+        <constraints nullable="true"/>
+      </column>
+      <column name="rp_time_requested" type="datetime">
+        <constraints nullable="true"/>
+      </column>
+      <column name="rp_time_reviewed" type="datetime">
```

Comment:
Based on yesterday's discussion, my guess is that reviews are going to be more complicated than just a single "yes/no" response, and there won't be a single time of review. (We may wind up either building or integrating with a ticketing system for this.)

Doesn't hurt to add it in for now, just food for thought.

Reply:
Agreed. I do sort of assume there's going to be some considerably more complicated thing than this.